core/workspace_comment_svg.js

```diff
@@ -132,7 +135,9 @@ Blockly.WorkspaceCommentSvg.prototype.dispose = function() {
  * @package
  */
 Blockly.WorkspaceCommentSvg.prototype.initSvg = function() {
-  goog.asserts.assert(this.workspace.rendered, 'Workspace is headless.');
+  if (!this.workspace.rendered) {
+    throw TypeError('Workspace is headless.');
```

Comment:
use Error

Reply:
I think this one is correct.  It is an error thrown when the workspace is of the wrong type, headless vs SVG.